src/main/java/at/hannibal2/skyhanni/features/inventory/wardrobe/WardrobeManager.kt

```diff
@@ -0,0 +1,43 @@
+package at.hannibal2.skyhanni.features.inventory.wardrobe
+
+import at.hannibal2.skyhanni.SkyHanniMod
+
+object WardrobeManager {
```

Comment:
The name feels weird, please use a different one

Reply:
updated